Makefile

```diff
@@ -61,3 +62,13 @@ pdk-phase-checks:
 	luacov -c t/phase_checks.luacov
 	grep "ngx\\." t/phase_checks.report
 	grep "check_" t/phase_checks.report
+
+fix-windows:
+	@for script in $(WIN_SCRIPTS) ; do \
+	  echo Converting Windows file $$script ; \
+	  mv $$script $$script.win ; \
+	  tr -d '\015' <$$script.win >$$script ; \
+	  rm $$script.win ; \
+	  chmod 0755 $$script ; \
```

Comment:
Was this needed btw? I never had an issue without it

Reply:
Yes. `>` produces a new file without the executable bit set.